src/include/ceph_fs.h

```diff
@@ -396,6 +396,7 @@ extern const char *ceph_mds_op_name(int op);
 #define CEPH_O_CREAT           00000100
 #define CEPH_O_EXCL            00000200
 #define CEPH_O_TRUNC           00001000
+#define CEPH_O_LAZY            00020000
```

Comment:
I'm okay with a function to enable LAZYIO (e.g. `ceph_lazyio`) and a blanket config but I think we should forgo this open flag `CEPH_O_LAZY`. It's not standard and because it was unusable (can't modify existing apps), LAZYIO never got traction years ago.